src/pages/git/git-pull/index.md

```diff
@@ -3,13 +3,31 @@ title: Git Pull
 ---
 ## Git Pull
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/git/git-pull/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+[**git pull**](https://git-scm.com/docs/git-pull) allows you to get new commits from a remote repository. To do this, *git pull* fetches ( *git fetch* ) the new commits and merges [(*git merge*)](https://guide.freecodecamp.org/git/git-merge) these into your local branch.
```

Comment:
For formatting of the `git ...` commands in this paragraph, let's put them in backticks for syntax highlighting instead of italics/bold. So **git pull** -> `git pull`, ( *git fetch* ) -> (`git fetch`), and (*git merge*) -> (`git merge`). Also, very minor nit, so the first word in the paragraph is capitalized, change "git pull allows you..." to "The `git pull` command allows you..."